### ISSUE TITLE
Measuring testing coverage

### ISSUE THREAD

**Issue by bquast** [r-hub/proposal#4]
If continuous integration is implemented, perhaps it would be useful to design this in such a way that it would also measure the extend to which testing covers the code, similar to:
- https://coveralls.io/
- https://codecov.io/

And congrats on your grant! the project looks amazing!

**Comment by gaborcsardi** [comment]
:+1: Thanks! Absolutely, we'll have code coverage.

**Comment by bquast** [comment]
Great, I look forward to using this

**Comment by jimhester** [comment]
cough https://github.com/jimhester/covr cough

@gaborcsardi codecov particularly is already supported for jenkins (relevant for r-hub) assuming the normal environment variables are set on the jenkins server.

**Comment by gaborcsardi** [comment]
Yeah, covr is the way to go. But we need some basic infrastructure first. :)
